src/ORM/Connect/DBSchemaManager.php

```diff
@@ -849,6 +861,29 @@ abstract public function IdColumn($asDbValue = false, $hasAutoIncPK = true);
      */
     abstract public function checkAndRepairTable($tableName);
 
+    /**
+     * Checks if we should be checking and repairing tables generally, and whether the passed in table
```

Comment:
A nitpick, but the double check sounded weird
```suggestion
     * Determines if we should be checking and repairing tables generally, and whether the passed in table
```

Reply:
Done